app/models/server/models/Users.js

```diff
@@ -1124,6 +1124,42 @@ Find users to send a message by email if:
 	getActiveLocalUserCount() {
 		return this.findActive().count() - this.findActiveRemote().count();
 	}
+
+	async findAllResumeTokensByUserId(userId) {
```

Comment:
instead of adding this function here and using `rawCollection()`, please consider creating this method on [UsersRaw](https://github.com/RocketChat/Rocket.Chat/blob/5d870988990ed6dfeb7e238dd3244fd9c6f2e7cf/app/models/server/raw/Users.js#L3) model.